src/commands/deployment/validate.ts

```diff
@@ -0,0 +1,67 @@
+import commander from "commander";
+import { logger } from "../../logger";
+import { config } from "../init";
+
+/**
+ * Adds the validate command to the commander command object
+ * @param command Commander command object to decorate
+ */
+export const validateCommandDecorator = (command: commander.Command): void => {
+  command
+    .command("validate")
+    .alias("v")
+    .description(
+      "Validate deployment(s) for a service, release environment, build Id, commit Id, or image tag."
+    )
+    .action(() => {
+      isValidConfig();
+    });
+};
+
+/**
+ * Validates that the deployment configuration is specified.
+ */
+export const isValidConfig = (): boolean => {
+  let missingConfig = "";
```

Comment:
instead of appending (and re-binding) to the same string for missing configs, could you use an array instead? ie

```
const missingConfig = [];

if (!config.deployment) {
  missingConfig.push("deployment")
} 

// other checks...

const missingConfigsMessage = missingConfig.join(' ');
...
```